azure-py-aks/__main__.py

```diff
@@ -0,0 +1,125 @@
+import pulumi
+from pulumi import ResourceOptions
+from pulumi_azure.core import ResourceGroup
+from pulumi_azure.role import Assignment
+from pulumi_azure.ad import Application, ServicePrincipal, ServicePrincipalPassword
+from pulumi_azure.containerservice import KubernetesCluster, Registry
+from pulumi_azure.network import VirtualNetwork, Subnet
+
+config = pulumi.Config('azure-py-kubernetes')
+PREFIX = config.get('prefix') or 'replaceme'
+PASSWORD = config.get('password') or 'DEf34tg!rg35y#F23'
+SSHKEY = config.get('sshkey') or 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCxinIAIDDCradZPAgX5GzBLv00u4rigOLUbU00E44FrfMTqu5wXiejJ4ycSb1bI+//ZNgaB2UYRbPL7A9OUKY+K4sX5O84Q6DPMjo/90IANHVTLf3xTaSc7hpvXOtIjJTJeiamxClgnTAcR55RV/j9/Wptxa8GGcRmRCcSmJUkx5AZTFI+s8aF0W3aeHHRw7TxNKBuwrX7FDcHyGKvdkFg4OP863Xe5hp5ql1C3XibmCOp1CMPIU2hCmGOy1LGbOf/Pa+QKAdtUSrPNK/jBWvPWo0k02Ii0JtMAdlpVqnJc3czNIp5gEqZCRCGEdkb/kZnJiMRZhmLBYnC8tiMxvZj core@k8s'
+LOCATION = config.get('location') or 'westeurope'
```

Comment:
I think this is not a real Azure region. Can we default to `East US`, or `config.require`?

Reply:
why not, it is a real azure region ;)

Reply:
i can switch to `eastus` if you want

Reply:
I actually would prefer it be in the US since most of our users are probably in the US, but it's surprising to me that `East US` and `eastus` are both valid regions. But, I am not well versed in Azure.